LayoutTests/TestExpectations

```diff
@@ -7501,3 +7501,71 @@ webkit.org/b/272417 imported/w3c/web-platform-tests/svg/path/property/priority.s
 
 # re-import css/css-align WPT failure
 webkit.org/b/271692 imported/w3c/web-platform-tests/css/css-align/blocks/align-content-block-break-overflow-020.html [ ImageOnlyFailure ]
+
+# Anchor positioning not yet implemented
+webkit.org/b/274926 imported/w3c/web-platform-tests/css/css-anchor-position/anchor-center-002.html [ ImageOnlyFailure ]
```

Comment:
Does it make sense to mark these as `Skip`, since they are not implemented?

Reply:
Makes sense. I will change them.

Reply:
I don't think we should be skipping these tests just because we don't implement the feature. We lose out on information when they start passing as you work on the feature and I don't think they are expensive enough to justify skipping.